src/components/UI/Text/Textarea.vue

```diff
@@ -194,10 +204,12 @@ export default {
     },
   },
   methods: {
-    inputChange(event) {
+    update() {
+      this.$emit("update:value", this.editor.getText() ? this.editor.root.innerHTML : "");
+    },
+    inputChange() {
       // invoked on input change
       this.$emit("input", this.value);
```

Comment:
This is not being used right now right? I guess this is in your todo list

Reply:
This inputchange method is in use but the emit method inside it is not in need ,I will remove that.